packages/python/src/components/TypeExpression.tsx

```diff
@@ -0,0 +1,71 @@
+import { Children, For, memo } from "@alloy-js/core";
+
+export interface SingleTypeExpressionProps {
+  children: Children;
```

Comment:
This one definitely doesn't seem like it should be `children` (in name or in type). It should at least be a `Child`, and perhaps more specifically a string or a ref?

Reply:
It's a convention to use Children in the library. Even for single items, it seems.

Let me check about making this more specific.